addons/sale_timesheet_margin/models/sale_order_line.py

```diff
@@ -23,10 +23,10 @@ def _compute_purchase_price(self):
             for line in timesheet_sols:
                 line = line.with_company(line.company_id)
                 product_cost = mapped_sol_timesheet_amount.get(line.id, line.product_id.standard_price)
-                if line.product_id.uom_id != line.company_id.project_time_mode_id and\
-                   line.product_id.uom_id.category_id.id == line.company_id.project_time_mode_id.category_id.id:
-                    product_cost = line.company_id.project_time_mode_id._compute_quantity(
+                if line.product_uom != line.company_id.project_time_mode_id and\
```

Comment:
Are we sure `product_uom` will be always defined? 🤔

Reply:
You are right, it may be not defined, I added as fallback the uom/category defined in product